modules/@apostrophecms/image/ui/apos/components/AposMediaManagerEditor.vue

```diff
@@ -145,7 +145,7 @@ export default {
       }
     }
   },
-  emits: [ 'saved', 'back', 'modified' ],
```

Comment:
why?

Reply:
No needed anymore, was fetching images twice: with `content-change` bus event and with `saved`